app/controllers/concerns/pages.rb

```diff
@@ -0,0 +1,39 @@
+module Pages
+  extend ActiveSupport::Concern
+
+  included do
+    resource_actions :default
+
+    schema_type :strong_params
+
+    allowed_params :create, :url_key, :title, :content, :language
+    allowed_params :update, :url_key, :title, :content, :language
+
+    before_filter :set_language_from_header, only: [:index]
+
+    define_method(:link_header) do |resource|
```

Comment:
Interesting, the inheritance provides the DSL to wire up the resource actions and thus the call the include createable resource code only happens when that DSL is run. So i can see that running after that call is made in the include block can work. Any ruby magic to avoid this? If not maybe leave a comment on those lines for future.